test/spec/data.wast

```diff
@@ -229,30 +229,30 @@
 (assert_trap
   (module
     (memory 1 2)
-    (data (i32.const 0x1_0000) "a")
+    (data (i32.const 65536) "a")
   )
   "out of bounds memory access"
 )
 (assert_trap
   (module
     (import "spectest" "memory" (memory 1))
-    (data (i32.const 0x1_0000) "a")
+    (data (i32.const 65536) "a")
```

Comment:
`0x10000` should work, and might be clearer? Also for `131072`

Reply:
Reverted as explained above.